python/taichi/lang/__init__.py

```diff
@@ -286,7 +316,24 @@ def init(arch=None,
          default_ip=None,
          _test_mode=False,
          **kwargs):
+    """Initializes the Taichi runtime.
+
+    This should always be the entry point of your Taichi program. Most
+    importantly, it sets the backend used throughout the program.
 
+    Args:
+        arch: Backend to use. This is usually :const:`~taichi.lang.cpu` or :const:`~taichi.lang.gpu`.
+        default_fp (Optional[type]): Default floating-point type.
+        default_ip (Optional[type]): Default integral type.
+        **kwargs: Taichi provides a large number of flags for configuration. Below we list a few
```

Comment:
maybe say this in another way? ;) Taichi provides highly customizable compilation through kwargs which allows fine grained control of Taichi compiler behavior.

Reply:
SG!